LICENSE.md

```diff
@@ -0,0 +1,9 @@
+MIT License
+
+Copyright (c) 2018 Richard Littauer
```

Comment:
I'm not sure this is the best approach, since you are not one of the main contributors according to the github insights page.